packages/modules/electricity_tariffs/octopusenergy/config.py

```diff
@@ -0,0 +1,18 @@
+from typing import Optional
+
+
+class OctopusEnergyTariffConfiguration:
+    def __init__(self, email: Optional[str] = None, accountId: Optional[str] = None, password: Optional[str] = None):
```

Comment:
In Zeile 12 hast Du fir Parameter für die Init-Funktion umgebrochen, dass fände ich hier auch schöner.

Reply:
done